### PR TITLE
employee dashboard with react component customer searching

### PR DESCRIPTION
**Pull request by dillonstreator**
Employee dashboard with react component for dynamic customer lookup table.

### CODE REVIEW

**Comment by dillonstreator** [comment]
There is a problem with the table in that the emails do not line up with the email column.  I wanted to push this anyway though so that others can review the code.

File: app/assets/javascripts/components/customer_search.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        <td className="col s3 center" key={data.id}>{data.id}</td>
+        <td className="col s3 center" key={data.email}>{data.email}</td>
+      </tr>
+    );
+    return (
+      <span>{row}</span>
+    );
+  }
+})
+
```
Comment (heydanhey) [inline app/assets/javascripts/components/customer_search.jsx:43]: Cool! I like how you broke that up into small components.  This is just a question, is there any reason these components `Table` and `TableRow` can't be in separate `.jsx` files?

**Review by heydanhey — APPROVED** [review]
+1, Great work!  We'll have to see how this behaves in production. Probably have to compile the assets and whatnot.

File: app/assets/javascripts/components/customer_search.jsx (hunk 1 above)
Comment (dillonstreator) [reply app/assets/javascripts/components/customer_search.jsx:43]: I'm able to split up the table and row into different components, but still have the problem with the row data not lining up with the column that it belongs to.

Comment (dillonstreator) [reply app/assets/javascripts/components/customer_search.jsx:43]: NVM... pull#105 seems to have fixed the issues :)
also split table up into multiple components

**Pull request merged**